safe-global/web-core
# feat: EIP-1271 accordion details

## What it solves

Resolves #1157

## How this PR fixes it

`MsgDetails` now renders the details of a message within the `MsgSummary` accordion, matching that of the transaction list style.

## How to test it

Open the "Messages" from the transaction list and expand the accordions. Observe the stubbed message details rendered in the same style as multisend transactions.

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/203043700-1d6a9d9f-6ca5-4c51-bb88-4319cd898aa5.png)

![image](https://user-images.githubusercontent.com/20442784/203043734-24113645-b923-4eb3-a98a-e7818890a0b5.png)
